test/lib/expr_test_utils.hh

```diff
@@ -98,6 +98,57 @@ std::pair<evaluation_inputs, std::unique_ptr<evaluation_inputs_data>> make_evalu
     const schema_ptr& table_schema,
     const column_values& column_vals,
     const std::vector<raw_value>& bind_marker_values = {});
+
+// A mock implementation of data_dictionary::database, used in tests
+class mock_database_impl : public data_dictionary::impl {
+    schema_ptr _table_schema;
+    ::lw_shared_ptr<data_dictionary::keyspace_metadata> _keyspace_metadata;
+
+    db::config _config;
+
+   public:
```

Comment:
bad indent.

Reply:
It's a new file so I took the chance to keep it compliant with `clang-format`.
Adding `AccessModifierOffset: -4` fixed this.